scripts/ci/docker-compose/local-prod.yml

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: "2.2"
+services:
+  airflow:
+    # We need to mount files an directories individually because some files
+    # such apache_airflow.egg-info should not be mounted from host
+    # we only mount those files that it makes sense to edit while developing
+    # or those that might be useful to see in the host as output of the
+    # tests (such as logs)
+    volumes:
+      - ../../../.bash_aliases:/root/.bash_aliases:cached
+      - ../../../.bash_history:/root/.bash_history:cached
+      - ../../../.github:/opt/airflow/.github:cached
+      - ../../../.inputrc:/root/.inputrc:cached
+      - ../../../.kube:/root/.kube:cached
+      - ../../../airflow:/opt/airflow/airflow:cached
```

Comment:
As discussed in #8229 -> we should not map the: "airflow, common, dags, requirements, scripts" in production image. They might be totally misleading and can have side effects if /opt/airflow is on the PYTHONPATH

Reply:
👍